docs/RELEASE.md

```diff
@@ -1,6 +1,35 @@
-# Release 0.4 - 12/6/2018
-
-## Major Features
+# ChangeLog
+
```

Comment:
please add v0.5 release...

Reply:
Actually I reverted the merge with v0.5 branch before initiating this PR. changes for v0.5 should not come in dev-doc branch currently. At least there should be a seperate merge PR.